coding-concepts.js

```diff
@@ -8,36 +8,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Bravo 2023"
-// console.log(cohort.split(""))
-
-// a) Your answer:
-// b) Verify and explain:
+//console.log(cohort.split(""))
+ 
+// a) Your answer: We're using the .split command so this should separate every letter of the given string to look like "B", "r", "a", "v" ...
+// b) Verify and explain: The built in method split will separate each character from the given string and put them into their own array. when switching an array to a string we would use .join, but in this case we are going from a string to an array so we will use .split. This will log the code [ "B", "r", "a", "v" ...]
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
-  `Hello, ${name}!`
+   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+//console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I think this code will log undefined because the return statement is not declared.
+// b) Verify and explain: The method as is logs undefined. if the desired oucome is Hello LEARN Student! then we would need to add the word return to line 19 before `Hello, ${name}!` Because the given code logged undefined, I suspected that the problem was in the functions return statement, or lack ther of.
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+//console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I originally thought this would log undefined because there is no return statement. It turns out this is refactored code and the higher order function .map will work as intended. This should return an array of every number multiplied by 2.
+// b) Verify and explain: This logs every number in the array by two. Because we used map the new array will be the same size as the original. We used the .map higher order function with a parameter of number. This is because the numbers from the array are what we want mapped, and multiplied by 2.
 
```

Comment:
It can certainly look misleading. Because it almost look likes a function. However the variable `multipliedByTwo` is storing the array created from running the map high order function ran on the array `[4, 5, 6, 7, 8]`